### PR TITLE
[feat] Integrate Doppler UI system header

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Change markup to adapt header with styles
- Use logic defined in library

### CODE REVIEW

**Comment by andresmoschini** [comment]
❤️

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -2,18 +2,35 @@ import React from 'react';
...
         <HeaderNav nav={headerData.nav} />
-        <HeaderUserMenu user={headerData.user} />
-      </header>
-    );
-  }
-}
+        <nav className="nav-right-main">
+          <ul>
+            <li>
+              <a href="http://google.com" className="active" data-count="1">
```
Comment (cbernat) [inline src/components/Header/Header.js]: Perhaps you could you change http://google.com for something more like doppler: https://www.fromdoppler.com/

Comment (andresmoschini) [reply src/components/Header/Header.js]: or maybe using _REACT_APP_API_URL_

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js [hunk 2]
```diff
@@ -4,42 +4,57 @@ const HeaderUserMenu = (props) => {
...
-        <div>
-          {user.plan.isSubscribers === 'true' || user.plan.isMonthlyByEmail === 'true' ? (
+      <span id="user-menu--open">
+        <span style={{ background: user.avatar.color }} className="user-avatar">
+          {user.avatar.text}
+        </span>
+      </span>
+      <div class="user-menu">
+        <header>
+          <span className="user-avatar--menu" style={{ background: user.avatar.color }}>
```
Comment (cbernat) [inline src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: Here you have an extra space `style={{ background: user.avatar.color }}` but in another properties you don't, this is only for style property? See below `{user.avatar.text}`

File: src/components/Header/Header.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Header/Header.js]: Fixed

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: Fixed

**Pull request merged**